src/render/theme.ts

```diff
@@ -6,7 +6,7 @@ export const theme = createMuiTheme({
     },
     props: {
         MuiButton: {
-            disableRipple: true
+            disableRipple: false
```

Comment:
You can just remove the entire `props` block. This merges with the base dark theme from MUI. Another cool application of the spread operator: `{ ...themeDefaults, ...themeOverrides }`